mandos-go/expression/interpreter/interpreter.go

```diff
@@ -181,6 +183,15 @@ func (ei *ExprInterpreter) InterpretString(strRaw string) ([]byte, error) {
 	return ei.interpretNumber(strRaw, 0)
 }
 
+// GetVMType yields the configured VM type, which is used for generating SC addresses.
+// Will yield default value [0, 0] is not explicitly configured.
+func (ei *ExprInterpreter) GetVMType() *[core.VMTypeLen]byte {
+	if ei.VMType == nil {
+		ei.VMType = &[core.VMTypeLen]byte{0, 0}
```

Comment:
I think this should yield WASM VM type by default.

Reply:
@andrei-marinica

Reply:
This component has no access to the VM. I made it editable, but there is no mechanism yet to set it.
There is a `var DefaultVMType = []byte{0xF, 0xF}`, should I rather have that as default?